### PR TITLE
docs(contrib): New contribution guidelines

### PR DESCRIPTION
**Pull request by connor-leech**
This is a work in progress. Do not merge/review.

Added stubs and left-nav items.

### CODE REVIEW

File: src/content/contributions/component/components.mdx [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+#### 2. Create a design spec
+
+Create sizing and styling annotations for all aspects of the component. This spec should provide a developer with everything they need to create the design in code. Check out our [production guidelines](https://github.com/carbon-design-system/carbon-website/wiki/Production-guidelines#spec-guidelines) to get started.
+
+You should include color tokens and type tokens used.
+
+<ImageComponent caption="Example of a spec" cols="8">
+
+![Example of a spec](images/spec.png)
```
Comment (aagonzales) [inline src/content/contributions/component/components.mdx]: This example is a little off, you wouldn't need that 2px spec which is measuring the spec not the component. Not a blocker to the PR but we could eventually drop a better example in here. Actually you could probably just pull an image from one of the component/style tabs as a good example.

File: src/content/contributions/component/components.mdx [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+</ImageComponent>
+
+#### 3. Create a Sketch symbol
+
+Any new components or changes to existing components will also live in the [Carbon Sketch kit](/getting-started/designers#get-the-kit) and so we'll need a Sketch symbol. Check out [Sketch's guide](https://www.sketch.com/docs/symbols/creating-symbols/) for creating a symbol. 
+
+This symbol can be contributed with the asset or enhancement, but must be added to the kit by one of its maintainers. You can read more about the role of maintainers in [Overview](/contributions/overview/overview#maintainers).
```
Comment (aagonzales) [inline src/content/contributions/component/components.mdx]: You can add they'll need to open an issue in the kit repo to have it added to the core kit.
https://github.com/carbon-design-system/carbon-design-kit/issues

File: src/content/contributions/pattern/pattern.mdx [hunk 3]
```diff
@@ -0,0 +1,58 @@
+---
+title: Patterns
+---
+
+### If you've designed a pattern that is not currently in Carbon, contributing it back allows others in the community to refine and use your pattern in their projects.
+
+## First steps to contributing
+
+The process for contributing a pattern will be depend on the pattern's complexity, but all contributions start by [opening a Github issue](https://github.com/carbon-design-system/carbon-website/issues/new). Include a detailed description in which you:
```
Comment (aagonzales) [inline src/content/contributions/pattern/pattern.mdx]: Typo: "The process for contributing a pattern will ~be~ depend on"

File: src/content/contributions/overview/governance.mdx [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
+We use what we build, and we've made it open source for anyone to use and contribute back to.
+
+### Who is the Code of Conduct for?
+
+The Carbon Design System is built for everybody and if you use or contribute to Carbon, you are part of this community and we expect you to adhere to our community standards.
+
+### Feedback and critique
+
+Building great products isn't easy, and there's usually more than one right answer. If you're taking part in any discussion, whether on Slack, Twitter, Gitter, GitHub, or even in person, please respect one another. Keep the mission in mind, and help us make Carbon better.
```
[the remark this answers is not in the document]
Comment (connor-leech) [reply src/content/contributions/overview/governance.mdx]: It's [Gitter](https://gitter.im/home/explore)!

File: src/content/contributions/component/components.mdx (hunk 1 above)
Comment (connor-leech) [reply src/content/contributions/component/components.mdx]: Will do. I just pulled this one from the production guidelines.

**Pull request merged**